src/Anagram.cpp

```diff
@@ -35,6 +42,14 @@ namespace
 
         return sorted;
     }
+
+    [[maybe_unused]] ListOfDicts::const_iterator findDictionary ( const std::string& dict )
```

Comment:
Is there ever a case where you'd call this and leave the result unused?

Reply:
It's there because of werror and the function was created to make sure the cmake worked.